packages/runtime/src/templates/edge/function-runtime.ts

```diff
@@ -11,6 +11,7 @@ const handler = async (req: Request, context: Context) => {
     return redirect
   }
   const request = buildNextRequest(req, context, nextConfig)
+  request.headers['x-matched-path'] ||= edgeFunctionDefinition.page
```

Comment:
Here, the `edgeFunctionDefinition.page` will populate the `x-matched-path` header only if it's not set, but from the issue this fixes, isn't the x-matched-path always not set, or is this because it's only when running on the edge in minimal mode?

Reply:
This is for forward-compatibility: edge router will pre-populate this value, so we'll keep that value if it's set